Qase_Test/Src/Steps/ApiSteps/ProjectApiSteps.cs

```diff
@@ -0,0 +1,27 @@
+using System.Net;
+using NUnit.Allure.Attributes;
+using Qase_Test.Models;
+using Qase_Test.Steps.ApiSteps.Base;
+using RestSharp;
+
+namespace Qase_Test.Steps.ApiSteps
+{
+    public class ProjectApiSteps : BaseApiStep
```

Comment:
better to call it ProjectApiHelper, it's a generally accepted decision. We should talk about it personally